bot/constants.py

```diff
@@ -236,39 +266,61 @@ def __str__(self) -> str:
     Month(Client.month_override)
 
 
-class Roles(NamedTuple):
+class _Roles(EnvConfig):
+
+    EnvConfig.Config.env_prefix = "roles_"
+
     owners = 267627879762755584
-    admins = int(environ.get("BOT_ADMIN_ROLE_ID", 267628507062992896))
+    admins = 267628507062992896
     moderation_team = 267629731250176001
-    helpers = int(environ.get("ROLE_HELPERS", 267630620367257601))
+    helpers = 267630620367257601
     core_developers = 587606783669829632
-    everyone = int(environ.get("BOT_GUILD", 267624335836053506))
+    everyone = Client.guild
+
+    lovefest = 542431903886606399
+
+
+Roles = _Roles()
+
 
+class _Tokens(EnvConfig):
+    EnvConfig.Config.env_prefix = "tokens_"
 
-class Tokens(NamedTuple):
-    giphy = environ.get("GIPHY_TOKEN")
-    aoc_session_cookie = environ.get("AOC_SESSION_COOKIE")
-    omdb = environ.get("OMDB_API_KEY")
-    youtube = environ.get("YOUTUBE_API_KEY")
-    tmdb = environ.get("TMDB_API_KEY")
-    nasa = environ.get("NASA_API_KEY")
-    igdb_client_id = environ.get("IGDB_CLIENT_ID")
-    igdb_client_secret = environ.get("IGDB_CLIENT_SECRET")
-    github = environ.get("GITHUB_TOKEN")
-    unsplash_access_key = environ.get("UNSPLASH_KEY")
+    giphy = ""
+    aoc_session_cookie = ""
+    omdb = ""
+    youtube = ""
+    tmdb = ""
+    nasa = ""
+    igdb_client_id = ""
+    igdb_client_secret = ""
+    github = ""
+    unsplash = ""
+    wolfram = ""
```

Comment:
Should these be `foo: str | None` instead? Maybe even pydantic secret strs?

Reply:
I'll make them nullable.
As for the `SecretStr`, see my comment [here](https://github.com/python-discord/sir-lancebot/pull/1260#discussion_r1185941910)